src/software/ai/navigator/path_planner/global_path_planner.cpp

```diff
@@ -0,0 +1,59 @@
+#include "software/ai/navigator/path_planner/global_path_planner.h"
+
+GlobalPathPlannerFactory::GlobalPathPlannerFactory(
```

Comment:
is the idea that we'd have multiple factories for different field sizes, like div_a and div_b?

Reply:
yes, this would be configured by passing in the relevant world